src/status_im/ui/screens/discover/views.cljs

```diff
@@ -96,6 +97,41 @@
                                          :current-account current-account}]])]
      [react/text (i18n/label :t/none)])])
 
+(def public-chats-mock-data
+  [{:name  "Status team"
+    :count 25
+    :color "#B2F3E3"}
+   {:name  "ETH news"
+    :count 12
+    :color "#F7A7E8"}
+   {:name  "All about Ethereum"
+    :count 32
+    :color "#C1B8F0"}])
+
+(defn render-public-chats-item [item]
+  [react/view styles/public-chats-item-container
+   [react/view styles/public-chats-icon-container
+    [react/view (styles/public-chats-icon (:color item))
+     [react/text {:style styles/public-chats-icon-text}
+                 (-> item :name first str)]]]
+   [react/view styles/public-chats-item-inner
+    [react/view styles/public-chats-item-name-container
+     ;TODO(goranjovic) lightgray intentionally hardcoded while only a teaser
```

Comment:
https://github.com/bbatsov/clojure-style-guide#comments two semicolons should be there by this doc